reference/7.5/Microsoft.PowerShell.Utility/Unblock-File.md

```diff
@@ -70,7 +70,7 @@ PS C:\> dir C:\Downloads\*PowerShell* | Unblock-File
 
 This command shows how to find and unblock PowerShell scripts.
 
-The first command uses the **Stream** parameter of the *Get-Item* cmdlet get files with the
+The first command uses the **Stream** parameter of the _Get-Item_ cmdlet get files with the
```

Comment:
Cmdlet names should be in code spans per our style guide.
```suggestion
The first command uses the **Stream** parameter of the `Get-Item` cmdlet get files with the
```

Reply:
Ok, so the original was wrong in this case. From MD perspective, it just pointed to the wrong character being used.